scripts/encode2ga4gh.py

```diff
@@ -0,0 +1,296 @@
+                             str(score), units])
+        quantOutfile.write("%s\n" % outline)
+
+
+def write_rnaseq_tables(analysisIds, description, annotationId, output_folder):
+    log("Writing rnaseq tables")
+    for analysisId in analysisIds:
+        # create analysis id folder
+        make_dir(os.path.join(output_folder, analysisId))
+
+        # output table
+        rnaSeqTable = os.path.join(output_folder, analysisId, "rnaseq.table")
+
+        # write rnaseq table
+        with open(rnaSeqTable, "w") as rnaQuantFile:
+            writeRNAQuant(rnaQuantFile, analysisId, description, annotationId)
+
+
+def write_counts_tables(data, output_folder):
+    log("Writing counts tables")
+    for analysisId, samstatsfile in data:
+        # output table
+        countsTable = os.path.join(output_folder, analysisId, "counts.table")
+
+        samstats = getSamstats(samstatsfile)
+        if samstats != {}:
+            print(samstats.keys())
+            # write mapping stats table
+            with open(countsTable, "w") as samOutfile:
+                writeSamstats(samOutfile, samstats, analysisId)
+
+
+def write_dist_tables(data):
+    log("Writing distribution tables")
+    for analysisId, distfile in data:
+        # output table
+        distTable = os.path.join(analysisId, "dist.table")
+
+        # write mapping distribution table
+        distribution = getDistribution(distfile)
+        with open(distTable, "w") as distOutfile:
+            writeDistribution(distOutfile, distribution, analysisId,
+                              distribution["mapped"])
+
+
+def write_expression_tables(data, annotationId, output_folder):
+    log("Writing gene expression tables")
+    for analysisId, quantfile in data:
+        # output table
+        expTable = os.path.join(output_folder, analysisId, "expression.table")
+
+        # write expression table
+        print("processing {}".format(analysisId))
+        with open(expTable, "w") as quantOutfile:
+            write_expression(analysisId, annotationId, quantfile, quantOutfile)
+
+
+def makeParser(usage):
+    """
+        Default values will download and parse 4 quantifications from the
+        ENCODE Evaluation dataset as an example set of test data.
+    """
+
+    parser = optparse.OptionParser(usage=usage)
+    parser.add_option("--dataset", dest="dataset")
+    parser.add_option("--description", dest="description")
+    parser.add_option("--annotation", dest="annotationId")
+    parser.add_option("--file_limit", type="int", dest="subset")
+
+    parser.set_defaults(dataset="ENCSR000AJW", annotationId="Gencodev16",
+                        description="RNAseq data from ENCODE evaluation",
+                        subset=4)
+
+    return parser
+
+
+def main(argv):
+
+    usage = "Usage: {} <data-folder>".format(argv[0])
+    if len(argv) != 2:
+        print(Color.red(usage))
+        sys.exit(1)
+
+    parser = makeParser(usage)
+    (options, args) = parser.parse_args(argv[1:])
+
```

Comment:
There's a lot of blank lines here... I tend to agree with the PEP8 point of view

> Use blank lines in functions, sparingly, to indicate logical sections.